frontends/mit-open/src/page-components/LearningResourceCardTemplate/LearningResourceCardTemplate.tsx

```diff
@@ -103,23 +95,162 @@ const LRCImage: React.FC<LRCImageProps> = ({
     variant === "column"
       ? { height: imgConfig.height }
       : { width: imgConfig.width, height: imgConfig.height }
+
   return (
-    <CardMedia
+    <CardMediaImage
       component="img"
-      className="ol-lrc-image"
+      variant={variant}
       sx={dims}
       src={resourceThumbnailSrc(resource.image ?? null, imgConfig)}
       alt=""
     />
   )
 }
 
-const variantClasses: Record<CardVariant, string> = {
-  column: "ol-lrc-col",
-  row: "ol-lrc-row",
-  "row-reverse": "ol-lrc-row-reverse",
+const LIGHT_TEXT_COLOR = "#8c8c8c"
+const SPACER = 0.75
+const SMALL_FONT_SIZE = "0.75em"
+
+const StyledCard = styled(Card)`
+  display: flex;
+  flex-direction: column;
+
+  // Ensure the resource image borders match card borders
+  .MuiCardMedia-root,
+  > .MuiCardContent-root {
+    border-radius: inherit;
+  }
+
+  .ol-lrc-details {
+    /*
+    Make content flexbox so that we can control which child fills remaining space.
+    */
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+
+    > * {
+      /*
+      Flexbox doesn't have collapsing margins, so we need to avoid double spacing.
+      The column-gap property would be a nicer solution, but it doesn't have the
+      best browser support yet.
+      */
+      margin-top: ${SPACER / 2}rem;
+      margin-bottom: ${SPACER / 2}rem;
+
+      &:first-of-type {
+        margin-top: 0;
+      }
+
+      &:last-child {
+        margin-bottom: 0;
+      }
+    }
+  }
+
+  .ol-lrc-chip.MuiChip-root {
+    height: 2.5 * ${SMALL_FONT_SIZE};
+    font-size: ${SMALL_FONT_SIZE};
+
+    .MuiSvgIcon-root {
+      height: 1.25 * ${SMALL_FONT_SIZE};
+      width: 1.25 * ${SMALL_FONT_SIZE};
+    }
+  }
```

Comment:
The multiplication here is invalid syntax. (This was compile-time scss syntax.) (I guess it should be `${ 2.5 * SMALL_FONT_SIZE_EM } em` now.

Reply:
Yes, that was not going to work - fixed. Unfortunately unitless values in Emotion object styles default to pixels. Generally, fixed pixel border and margin styles are more likely to preserve a good layout if the user increases font scaling as where we increase container width and height relative to default font size, we introduce extra space and reduce the available space for text, also not accounting for wrapping.